### ISSUE TITLE
Show node's Eth address by default in Terminal

### ISSUE THREAD

**Issue by chrishobcroft** [livepeer/go-livepeer#438]
# Current Situation

When running livepeer with no arguments i.e. default settings by running `./livepeer`, the application does not show the node's Ethereum address.

See below:

![image](https://user-images.githubusercontent.com/2212651/40361993-7df33d20-5dcb-11e8-82b8-1e893afe75c2.png)

# Further Context

I am aware that if you add the flag `-v 6` then this gives more verbose logging, which _does_ include the Eth address, but it also shows a lot of distracting and confusing information which is not relevant.

See below:

![image](https://user-images.githubusercontent.com/2212651/40362167-0f37c0c6-5dcc-11e8-8ca1-84f5bdfc5395.png)

# Proposed Change

This proposal is to change the code to print the node's Eth address to the screen, so that the user is aware of the Eth address, without needing to open `livepeer_cli`, and without needing to add `-v 6` flag.

Specific proposal is to show the following text when starting up a node using only `./livepeer` command:

```
chrishobcroft@livepeertv:~/Desktop/livepeer_linux$ ./livepeer
I0522 14:29:45.724988    7586 livepeer.go:144] ***Livepeer is running on the Ethereum main network: 0xf96d54e490317c557a967abfa5d6e33006be69b3***
I0522 14:24:29.008741    7393 accountmanager.go:58] Using ETH account: 0xe1C879F065682A0d35966596A72d5681a56118A5
I0522 14:24:29.810151    7393 accountmanager.go:76] Passphrase required to unlock ETH account
Passphrase: 
```

...where the `Using ETH account:...` line is new and shown by default.

**Comment by chrishobcroft** [comment]
Hey @yondonfu, I don't know if this has been released with `0.2.4` but I just checked and it isn't showing the node's Eth address by default in Terminal...

This is what I saw:

![image](https://user-images.githubusercontent.com/2212651/41937057-8645b50e-798f-11e8-8d71-b577d18e7f61.png)

**Comment by yondonfu** [comment]
I believe this just missed the `0.2.4` release, but it'll be included in the next release.
